src/components/TestComponent.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React from 'react';
+import propTypes from 'prop-types';
+import connectComponent from '../lib/connect-component';
+import checkIsReducing from '../actions/initialAction';
+
+
+export const TestComponent = (props) => {
+  const { initialReducer: { isReducing }, dispatch } = props;
+  return (
+    <>
+      <h3>If redux is working, the value should toggle between true and false</h3>
+      <h4>
+        Hey! I am
+        {' '}
+        {isReducing}
+      </h4>
+      <button type="submit" onClick={dispatch}>
+        Show
+        {' '}
+        {isReducing === 'true' ? 'false' : 'true'}
```

Comment:
when isReducing is changed to a boolean, this can be reduced to `{isReducing && 'false'}`

Reply:
Rather you can do this instead.
```
<button type="submit" onClick={dispatch} disabled={isReducing}>
```

Reply:
This would disable disable the button of isReducing were to have a boolen value. But the implementation from the screen shot is to make the button's child show true if the value of isReducing is a string with a value of false. The string value was returned as such because a boolean would not display o the DOM.